README.md

```diff
@@ -110,3 +113,32 @@ RULES = [
     ),
 ]
 ```
+
+## Container Usage ##
+
+To use the container image you first have to build it.  
+This assumes you have `docker` installed on your system.  
+In case you have setup your Artifactory with self-signed certificates, place all certificates of the chain of trust into the `container/certificates/` folder.  
```

Comment:
Any details should be placed after "common usage" section, the first thing people want to see - a bash command how to run the docker. 
Let's move self-signed certificated info to the bottom of this section

`container/certificates/` shouldn't it be `docker/certificates`? 

It there a way how to add these certificates in `docker run` command not in `build`? It'd be better